setupext.py

```diff
@@ -1817,8 +1811,6 @@ def add_flags(self, ext):
             (tcl_lib_dir, tcl_inc_dir, tcl_lib,
              tk_lib_dir, tk_inc_dir, tk_lib) = result
             ext.include_dirs.extend([tcl_inc_dir, tk_inc_dir])
-            ext.library_dirs.extend([tcl_lib_dir, tk_lib_dir])
-            ext.libraries.extend([tcl_lib, tk_lib])
```

Comment:
Do we need any of this stuff (including finding header files) anymore if we've duplicated the function signatures (as function pointer signatures) in our own code now anyway?